apps/aehyperchains/src/aehc_connector.erl

```diff
@@ -0,0 +1,85 @@
+%%% -*- erlang-indent-level: 4 -*-
+%%%-------------------------------------------------------------------
+-module(aehc_connector).
+
+-export([connector/0]).
+
+-export([send_tx/1, get_block/1]).
+-export([tx/2, block/4]).
+-export([publish_block/1, subscribe_block/0]).
+
+-type connector() :: atom().
+
+-callback send_tx(binary()) -> binary().
+
+-callback get_block(non_neg_integer()) -> block().
+
+%%%===================================================================
+%%%  Parent chain simplified proto
+%%%===================================================================
+
+-record(tx, { sender_id :: binary(), payload :: binary() }).
```

Comment:
Looks good for now. This will be adjusted slightly after the DB api is merged

Reply:
Sure, I will track that